ui/src/shared/components/EditableName.tsx

```diff
@@ -15,7 +15,7 @@ import 'src/shared/components/EditableName.scss'
 interface Props {
   onUpdate: (name: string) => void
   name: string
-  onEditName: () => void
+  onEditName: (e?: any) => void
```

Comment:
might be better to use a ChangeEvent here

Reply:
I agree, I will fix it in next pr.